src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/Json/XmlObjectSerializerReadContextComplexJson.cs

```diff
@@ -10,20 +10,25 @@
 using System.Runtime.Serialization.Json;
 using System.Runtime.Serialization;
 using DataContractDictionary = System.Collections.Generic.Dictionary<System.Xml.XmlQualifiedName, System.Runtime.Serialization.DataContract>;
+#if !NET_NATIVE
+using ExtensionDataObject = System.Object;
+#endif
 
 namespace System.Runtime.Serialization.Json
 {
 #if NET_NATIVE
     public class XmlObjectSerializerReadContextComplexJson : XmlObjectSerializerReadContextComplex
+#elif MERGE_DCJS
```

Comment:
Why this class needs to be internal in NetCore? Can it be public? Or can it be internal in NetNative?